test/Utils/LoggingSanitizer.spec.ts

```diff
@@ -19,44 +19,66 @@ describe("LoggingSanitized unit tests", () => {
           longitude: "",
           latitude: ""
         };
+
+        const configObject = {
+          method: "get",
+          headers: {
+            Accept: "application/json, text/plain, */*",
+            AuthenticatedUserToken: "ew0KIsfqYWxnIjogIlJTMjU2IiwNCiAgInR5cCI6ICJKV1QiLA0KICAia2lkIjogIlpTdS8zcG1LWjJpSStmQmN6STkzU002Mm9VWmkvbkV5Zys4a0ZWaWkyVWs9Ig0KfQ.ew0KICAiaXNzIjogIjYzN2ZjYzlmLTRhOWItNGFhYS04NzEzLWEyYTNjZmRhMTUwNSIsDQogICJpYXQiOiAxNjI0ODcxMjA0LA0KICAibmJmIjogMTYyNDg3MTIwNCwNCiAgImV4cCI6IDE2MjQ5NTc2MDQsDQogICJzdWIiOiAiYjYyNTIyYjktNzI3ZC00MGE3LWI1ODgtOTk3OWQxM2Q3Mzc2IiwNCiAgIm9pZCI6ICJiNjI1MjJiOS03MjdkLTQwYTctYjU4OC05OTc5ZDEzZDczNzYiLA0KICAidGlkIjogImQ2OWUxZTkwLWZiZTUtNGI3YS1iNGI0LTVjZDM3NzdlMmVhMyIsDQogICJ1cG4iOiAiVENTQWRtaW5AVElQQWRtaW5BUEkub25taWNyb3NvZnQuY29tIiwNCiAgIkZpcnN0TmFtZSI6ICJBZG1pbiIsDQogICJMYXN0TmFtZSI6ICJUQ1MiLA0KICAibHdpY29udGV4dHMiOiB7DQogICAgIkNhc2VOdW1iZXIiOiB7DQogICAgICAidmFsdWUiOiAiMjEwNjI4MDA1MDAwMDkxOCIsDQogICAgICAiaXNEaXNwbGF5YWJsZSI6IHRydWUNCiAgICB9LA0KICAgICJDYXNlVGl0bGUiOiB7DQogICAgICAidmFsdWUiOiAiVGhpcyBpcyBhIHRlc3QgY2FzZSBjcmVhdGVkIGJ5IE1pY3Jvc29mdCBpbnRlcm5hbCB0ZXN0aW5nLiBQbGVhc2UgY2xvc2UgaW1tZWRpYXRlbHkgYW5kIG1ha2Ugc3VyZSB0byBzZXQgdGhlIGNsb3NlIHN0YXR1cyBhcyBEdXBsaWNhdGUiLA0KICAgICAgImlzRGlzcGxheWFibGUiOiB0cnVlDQogICAgfSwNCiAgICAiQWN0aXZlU3lzdGVtIjogew0KICAgICAgInZhbHVlIjogIkR5bmFtaWNzIiwNCiAgICAgICJpc0Rpc3BsYXlhYmxlIjogdHJ1ZQ0KICAgIH0sDQogICAgIkZpcnN0TmFtZSI6IHsNCiAgICAgICJ2YWx1ZSI6ICJBZG1pbiIsDQogICAgICAiaXNEaXNwbGF5YWJsZSI6IHRydWUNCiAgICB9LA0KICAgICJMYXN0TmFtZSI6IHsNCiAgICAgICJ2YWx1ZSI6ICJUQ1MiLA0KICAgICAgImlzRGlzcGxheWFibGUiOiB0cnVlDQogICAgfSwNCiAgICAiQ3VycmVudFVzZXJFbWFpbCI6IHsNCiAgICAgICJ2YWx1ZSI6ICJ2LWFrZGhvdUBtaWNyb3NvZnQuY29tIiwNCiAgICAgICJpc0Rpc3BsYXlhYmxlIjogdHJ1ZQ0KICAgIH0sDQogICAgIlByb2R1Y3ROYW1lIjogew0KICAgICAgInZhbHVlIjogIlBvd2VyIEF1dG9tYXRlIiwNCiAgICAgICJpc0Rpc3BsYXlhYmxlIjogdHJ1ZQ0KICAgIH0sDQogICAgIkNhdGVnb3J5Ijogew0KICAgICAgInZhbHVlIjogIkFwcHJvdmFscyIsDQogICAgICAiaXNEaXNwbGF5YWJsZSI6IHRydWUNCiAgICB9LA0KICAgICJTdWJDYXRlZ29yeSI6IHsNCiAgICAgICJ2YWx1ZSI6ICJXb3JraW5nIHdpdGggYXBwcm92YWwgZmxvd3MiLA0KICAgICAgImlzRGlzcGxheWFibGUiOiB0cnVlDQogICAgfSwNCiAgICAiU2V2ZXJpdHkiOiB7DQogICAgICAidmFsdWUiOiAiU2V2ZXJpdHlDIiwNCiAgICAgICJpc0Rpc3BsYXlhYmxlIjogdHJ1ZQ0KICAgIH0sDQogICAgIlNlcnZpY2VMZXZlbCI6IHsNCiAgICAgICJ2YWx1ZSI6ICJQcm9mZXNzaW9uYWwiLA0KICAgICAgImlzRGlzcGxheWFibGUiOiB0cnVlDQogICAgfSwNCiAgICAiTGFuZ3VhZ2UiOiB7DQogICAgICAidmFsdWUiOiAiZW4tVVMiLA0KICAgICAgImlzRGlzcGxheWFibGUiOiB0cnVlDQogICAgfSwNCiAgICAiTG9jYXRpb24iOiB7DQogICAgICAidmFsdWUiOiAiVVMiLA0KICAgICAgImlzRGlzcGxheWFibGUiOiB0cnVlDQogICAgfQ0KICB9DQp9.PPEtQu9Q5ZfPPXJKdkbm2s6y71sXaHKXect602--Ow2sLUtt6rzkUngmSyVCIRe-Vn39k5N5jDyNqhB21PeqXbBM2JrVBZEB59c98wM-6TUUJYfJkdWeCRBqW_35UhosMlzijYcL3yxacFxh2I0TMQnwpzYrh3QTtwAam1mnUsXJ2J8IeZrNSutSc34c7_g4u94mmW3gG7xjw22ARscOsqLiYINXeVBzj-DCdt3L9IY6xsjDl3ihGan2X6ECLoy6KMMIxsBfbGWAsBhZjwUTH3fZL6n5sb9Z60gKf_q2oI6tyrnCJAfG8WkyfrYukYpOUZZMxJE_l2liiRhInbMsJg"
```

Comment:
Is it necessary to use a real token? Or any string is fine for testing. Just want to make sure we don't accidentally spill any credentials

Reply:
The token has been there since sanitizing existed. I verifed in the content that it does not contain any sensitive data. Regardless, I'll replace it with a mocked data.